core/crypto/key.go

```diff
@@ -10,7 +10,73 @@
 
 package crypto
 
-import "filippo.io/age"
+import (
+	"strings"
+	"sync"
+
+	"filippo.io/age"
+)
+
+const BlankRootKeyValue = "{}"
+
+// RootKey is the key used for encryption, decryption, backup, and restore.
+var RootKey = ""
+var RootKeyLock sync.RWMutex
```

Comment:
In retrospect, we can store this as an instance of `RootKeyTriplet` instead of a plain string.